playbooks/Installation/logging_with_fluentd.adoc

```diff
@@ -131,6 +136,61 @@ cat << EOF > /etc/td-agent/config.d/kubernetes.conf
 EOF
 ----
 
+OSE does not have logs of it's own, however if you are using an HA OSE environment there are some services that are useful to provide centralized logging.
```

Comment:
Can you change this to say something like openshift doesn't log to a unique file, but instead to /var/log/whatever (messages, i think). This makes it sound like openshift components don't log anything.